cmd/operator-sdk/generate/csv.go

```diff
@@ -74,14 +69,28 @@ Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config
 		},
 	}
 
-	cmd.Flags().StringVar(&c.csvVersion, "csv-version", "", "Semantic version of the CSV")
+	cmd.Flags().StringVar(&c.csvVersion, "csv-version", "",
+		"Semantic version of the CSV")
 	if err := cmd.MarkFlagRequired("csv-version"); err != nil {
 		log.Fatalf("Failed to mark `csv-version` flag for `generate csv` subcommand as required: %v", err)
 	}
 	cmd.Flags().StringVar(&c.fromVersion, "from-version", "",
 		"Semantic version of an existing CSV to use as a base")
-	cmd.Flags().StringVar(&c.csvConfigPath, "csv-config", "",
-		"Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml")
+	cmd.Flags().StringToStringVar(&c.inputs, "inputs",
+		map[string]string{
+			gencatalog.DeployDirKey: "deploy",
+			gencatalog.APIsDirKey:   filepath.Join("pkg", "apis"),
+		},
+		`Key value input paths used in CSV generation.
+Use this to set custom paths for operator manifests and API type definitions
+E.g: --inputs deploy=config/production,apis=pkg/myapp/apis 
```

Comment:
WDYT about use the examples as [here](https://github.com/operator-framework/operator-sdk/blob/ad1d4d1953abf22c336f67198fc23a489c0f4fdb/cmd/operator-sdk/add/controller.go#L45)

Reply:
Not in the help text of the flag itself but I can add an example in the examples section of the cmd.